docs/user/usage.md

```diff
@@ -12,12 +12,13 @@ After having [installed the requirements and built](../installation/build.md) sp
 
 The docker image can also be used directly, as detailed in the specific [docker documentation page](docker.md).
 
-The Nano S is the default model; the Nano X and Blue can be specified on the
+The Nano S is the default model; the Nano X, Stax and Blue can be specified on the
```

Comment:
Technically, on applications built from recent SDK, the model is embedded in the elf and detected by speculos, so there is no needed to specify it anymore.

Reply:
Thanks for the info. I added it.